### PR TITLE
Address bugs

### LINKED ISSUE #256: [IMPROVE] Loading Non US Stock Tickers
**What's the feature that should be improved?**
I believe that changing the default source or adding a feature that allows for loading for non us tickers would massively help in the outreach of the terminal.

**Describe how you would like the feature improved**
When I try to load Indian stock tickers like SBIN , BAJAJFINSV , EICHERMOT it is unable to load the required data.

**Possibly describe the ideal way to improve this**
I believe that adding an API extension that allows for Indian market data would solve the issue. I tried searching for one and found this https://nsetools.readthedocs.io/en/latest/introduction.html . Maybe, this would be of any help...I hope.

If you guys find a solution test it out with the following tickers SBIN , BAJAJFINSV , EICHERMOT , HDFCBANK . These are all tickers of companies listed in NSE.

### LINKED ISSUE #265: How to update?[IMPROVE]
**What's the feature that should be improved?**
I see there are frequent updates from you & team, what's the right way to update? As doing everything all over everything time is quite time-consuming. Thanks

**Describe how you would like the feature improved**
A description of what the current feature is vs what it would be after your suggestion.

**Possibly describe the ideal way to improve this**
If you have thought about how you would do it, add it here.

**Additional information**
Add any other information or screenshots about the feature improvement.

### PR DESCRIPTION
**Pull request by DidierRLopes**
Refactor main README, and add `update` section. Fixes #265

Added a comment for Indian tickers working with ".NS" at end, which closes #256 

Fixed the "view" cmd for indian tickers, because it was re-loading tickers from Alpha Vantage.

Refactored loading intraday data from yfinance because I found out that it's possible, and also load before/after market hours

### CODE REVIEW

File: gamestonk_terminal/main_helper.py [hunk 1]
```diff
@@ -117,7 +119,8 @@ def load(l_args, s_ticker, s_start, s_interval, df_stock):
     parser = argparse.ArgumentParser(
         add_help=False,
         prog="load",
-        description=""" Load a stock in order to perform analysis.""",
+        description=""" Load a stock in order to perform analysis.
+                        To load an Indian ticker use '.NS' at the end, e.g. 'SBIN.NS'. """,
```
Comment (jmaslek) [inline gamestonk_terminal/main_helper.py:123]: Should we specify the indian ticker?  For example, french tickers end in .PA, and I am sure there are many others. (Unless the issue was specifically .NS)

**Review by jmaslek — COMMENTED** [review]
looks good!  Was the Indian Tickers a specific error? Since I can load (example `ML.PA` (french) ) no problem, not sure if specifying one opens the floodgates for issues regarding certain markets.

File: README.md [hunk 2]
```diff
@@ -223,6 +220,30 @@ If you get the error that `statsmodels` is not found, you would run
 
 Then please submit an issue so that we can address why that was not imported.
 
+Please note that the package `pmdarima` needs to installed through `pip install` and not through `conda install`.
+
+### Update Terminal
+
+The terminal is constantly being updated with new features and bug fixes, hence, for your terminal to be updaate, you can run:
+```
+git pull
```
Comment (jmaslek) [inline README.md:229]: spelling on updaate

Comment (jmaslek) [reply README.md:229]: also should we specify that you need to be in the right directory?

File: gamestonk_terminal/main_helper.py (hunk 1 above)
Comment (DidierRLopes) [reply gamestonk_terminal/main_helper.py:123]: Yea, at least 3/4 people requested this. But you gave me a good idea, I'll add this link:
https://help.yahoo.com/kb/exchanges-data-providers-yahoo-finance-sln2310.html

File: README.md (hunk 2 above)
Comment (DidierRLopes) [reply README.md:229]: Good point, I'll do that. Thanks!

Comment (DidierRLopes) [reply README.md]: Regarding the directory, I think you can pull from anywhere, even if you are in sub-folders.

**Review by jmaslek — APPROVED** [review]

**Pull request merged**